nucypher/policy/policies.py

```diff
@@ -171,23 +176,42 @@ class Policy(ABC):
     An edict by Alice, arranged with n Ursulas, to perform re-encryption for a specific Bob.
     """
 
-    POLICY_ID_LENGTH = 16
+    ID_LENGTH = POLICY_ID_LENGTH
```

Comment:
Why do we have an additional constant here? Could it be just `ID_LENGTH = 16`, and whoever needs it would refer to it as `Policy.ID_LENGTH`?

Reply:
Disregard that, see my comment below.